### PR TITLE
[677<677-1] Add overview page for import

### PR DESCRIPTION
**Pull request by vpavicic**
### Description

Please briefly explain the changes you made here.

### Pull request type

Please delete options that are not relevant and check the ones that are.

- [ ] Documentation fix
- [ ] New documentation page 
- [ ] Base PR for the release
- [x] Documentation improvements
- [ ] Other (please describe):

### CODE REVIEW

File: docs/import-data/overview.md [hunk 1]
```diff
@@ -5,57 +5,80 @@ sidebar_label: Import data overview
...
 
-## How to migrate data from a relational database?
+## JSON files
 
-You can migrate data from a [**MySQL**](/import-data/migrate/mysql.md) or
-[**PostgresSQL**](/import-data/migrate/postgresql.md) database using the
-[**`mgmigrate`** tool](https://github.com/memgraph/mgmigrate).
+You can import the JSON file into Memgraph by using the [**`json-util` query
+module**](/import-data/files/load-json.md), which has procedures for loading JSON
+file from a local file  from a remote address.
```
Comment (katarinasupe) [inline docs/import-data/overview.md:38]:
```suggestion
file from a local file and from a remote address.
```

**Review by katarinasupe — CHANGES_REQUESTED** [review]
I see that you ditched the native/client idea, but I still like it. I think it is a good starting point and we can iterate later if we notice something that can be changed. Btw. some of the links are bolded, some are not, maybe unify that?

**Comment by katarinasupe** [comment]
Forgot to mention, one thing missing here - tip for fastest import. You can do similar to what I wrote in GQLA for now, and we will expand LOAD CSV subpage with ANALYTICAL MODE. https://github.com/memgraph/docs/pull/841/files#diff-20ae718e9607aa84472b93127370f5721e866009efb54b937fdb805d4738e872R19

File: docs/import-data/overview.md [hunk 2]
```diff
@@ -5,57 +5,80 @@ sidebar_label: Import data overview
...
+
+You can also use [`import_util.json` procedure](/import-data/files/load-json.md)
```
Comment (vpavicic) [inline docs/import-data/overview.md:40]:
```suggestion
You can also use [**`import_util.json` procedure**](/import-data/files/load-json.md)
```

File: docs/import-data/overview.md [hunk 3]
```diff
@@ -5,57 +5,80 @@ sidebar_label: Import data overview
...
+## Data from an application or a program
 
 Memgraph offers a wide range of drivers that can be used to connect directly to
 the platform. You can find a list of all the supported drivers
 [here](/connect-to-memgraph/drivers/overview.md).
 
+## Parquet, ORC or IPC/Feather/Arrow file
+
+If you are a Python user you can import Parquet, ORC or IPC/Feather/Arrow file
+into Memgraph [using GQLAlchemy](/gqlalchemy/how-to-guides/table-to-graph-importer).
```
Comment (vpavicic) [inline docs/import-data/overview.md:67]:
```suggestion
If you are a Python user you can import **Parquet**, **ORC** or **IPC/Feather/Arrow** file
into Memgraph [using **GQLAlchemy**](/gqlalchemy/how-to-guides/table-to-graph-importer).
```

File: docs/import-data/overview.md [hunk 4]
```diff
@@ -5,57 +5,80 @@ sidebar_label: Import data overview
...
+
+You can migrate data from a [**MySQL**](/import-data/migrate/mysql.md) or
+[**PostgresSQL**](/import-data/migrate/postgresql.md) database using the
+[**`mgmigrate`** tool](https://github.com/memgraph/mgmigrate).
+
+## Data from an application or a program
 
 Memgraph offers a wide range of drivers that can be used to connect directly to
 the platform. You can find a list of all the supported drivers
 [here](/connect-to-memgraph/drivers/overview.md).
```
Comment (vpavicic) [inline docs/import-data/overview.md:62]:
```suggestion
Memgraph offers a [**wide range of clients**](/connect-to-memgraph/drivers/overview.md) that can be used to connect directly to the platform and import data.
```

File: docs/import-data/overview.md [hunk 5]
```diff
@@ -5,57 +5,80 @@ sidebar_label: Import data overview
...
 within a **CYPHERL** file it can be [imported via Memgraph
 Lab or mgconsole](/import-data/files/cypherl.md).
 
-## How to migrate data from a relational database?
+## JSON files
 
-You can migrate data from a [**MySQL**](/import-data/migrate/mysql.md) or
-[**PostgresSQL**](/import-data/migrate/postgresql.md) database using the
-[**`mgmigrate`** tool](https://github.com/memgraph/mgmigrate).
+You can import the JSON file into Memgraph by using the [**`json-util` query
```
Comment (vpavicic) [inline docs/import-data/overview.md:36]:
```suggestion
You can import the **JSON** file into Memgraph by using the [**`json-util` query
```

**Comment by katarinasupe** [comment]
Fix broken links and ship it 🚀

**Pull request merged**